funding/manager.go

```diff
@@ -798,9 +798,13 @@ func (f *Manager) rebroadcastFundingTx(c *channeldb.OpenChannel) {
 	}
 }
 
+// PendingChanID is a type that represents a pending channel ID. This might be
+// selected by the caller, but if not, will be automatically selected.
+type PendingChanID = [32]byte
```

Comment:
Why not just use ChanID? afaict they are representationally equivalent and because you went for a type alias (as opposed to a wrapper type) this doesn't create any additional domain type safety.

Reply:
Alias was useful in `tapd`, so we can reference this value. `ChanID` is derived from a `wire.OutPoint`, while this is just a random value.